pycheribuild/config/target_info.py

```diff
@@ -860,6 +860,9 @@ def is_native(self) -> bool:
         assert self.target_info_cls is not None
         return self.target_info_cls.is_native()
 
+    def is_native_hybrid(self) -> bool:
```

Comment:
Maybe we could use `is_native_libcompat()`? That would allow using it for e.g. lib32 with -m32 to test this on non-Morello HW.